### PR TITLE
[FSK-63]: Cors Configuration

### PR DESCRIPTION
**Pull request by pourdaavar**

### CODE REVIEW

File: src/app.module.ts [hunk 1]
```diff
@@ -37,6 +37,12 @@ const mocks = {
       driver: ApolloDriver,
       playground: false,
       debug: true,
+      cors: {
+        // TODO: origin should be moved into configModule - not sure regex is valid!
+        // https://www.debuggex.com/r/XTrC-yG2Yeq2_sAm
+        origin: /https?:\/\/[a-z0-9\-_]+-fullstacks\.vercel\.app/,
+        credentials: true,
```
Comment (mhmda-83) [inline src/app.module.ts]: cause we don't use cookie, credentials could be false.

**Review by mhmda-83 — CHANGES_REQUESTED** [review]

File: src/app.module.ts (hunk 1 above)
Comment (MohammadForoutan) [reply src/app.module.ts]: @ASafaeirad   
do you think it's a good idea to have regex for origin?  
maybe someone creates an app with a similar domain

Comment (pourdaavar) [reply src/app.module.ts]: Based on [Access-Control-Allow-Credentials](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Credentials) explanations:
> Credentials are cookies, authorization headers, or TLS client certificates.

Should the authorization header send cross requests? if there is no auth header `credentials` is useless

Comment (mhmda-83) [reply src/app.module.ts]: Oh, i didn't know that (which Authorization header counts too)

**Review by mhmda-83 — APPROVED** [review]

**Review by ASafaeirad — CHANGES_REQUESTED** [review]
Please update the branch

**Comment by pourdaavar** [comment]
> 

done!

**Review by ASafaeirad — APPROVED** [review]

**Pull request merged**